bors.toml

```
@@ -0,0 +1,20 @@
+status = [
+  "test (1, ubuntu-latest, x86, 2)",
+  "test (1.%, ubuntu-latest, x64, 1)",
+  "test (1.%, ubuntu-latest, x64, 2)",
+  "test (1, ubuntu-latest, x64, 1)",
+  "test (1, ubuntu-latest, x64, 2)",
+  "test (1, macOS-latest, x64, 2)",
+  "test (1, windows-latest, x64, 2)",
+  "Turing.jl",
+  "format",
+  "docs"
```

This should be a list of the names of the Github actions that are required to pass (`1.%` is a regex for the oldest supported Julia version, otherwise one would have to update it from eg. `1.0` to `1.1` manually):
```suggestion
  "test (1.%, ubuntu-latest, x64)",
  "test (1.%, macOS-latest, x64)",
  "test (1.%, windows-latest, x64)",
  "test (1, ubuntu-latest, x64)",
  "test (1, macOS-latest, x64)",
  "test (1, windows-latest, x64)",
  "DynamicPPL.jl",
  "format"
```